src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/SendDailyReportTask.java

```diff
@@ -77,9 +78,10 @@ public void sendReport() {
                 recipients,
                 Map.of(getReportAttachmentName(reportDate), reportFile)
             );
+            logger.info("Finished {} job", TASK_NAME);
```

Comment:
should this not be after catch?

Reply:
Do not  think so - this indicates success

Reply:
Those logs are about start and finish and not to indicate success ? So if you have an exception you would never see finished ?

Reply:
Ok, will update @nitinprabhu

Reply:
success/failure for these tasks should be indicated with request/dependency feature by app insights. if ever this functionality comes in. on the other hand it can replace querying logs for some alerts. which is absurd